### DISCUSSION TITLE
DevSecOps

### DISCUSSION THREAD

**Discussion by BakangMonei** [community/community#140737] [Programming Help]
### Body

What strategies have you found effective in integrating security (DevSecOps) into your CI/CD pipeline without significantly slowing down development velocity?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/140735#discussioncomment-10873693) for reference to why this discussion was closed and locked. Thanks!
